src/app/for-nonprofits/page.tsx

```diff
@@ -0,0 +1,112 @@
+"use client";
+import "@/decisions-card/DecisionCriteriaCard";
+import { Container, Image, SimpleGrid, Text, Title, Stack } from "@mantine/core";
+import Banner from "@/components/Banner/Banner";
+import "@mantine/core/styles.css";
+import Footer from "@/components/Footer/Footer";
+import { DecisionCard } from "@/decisions-card/DecisionCriteriaCard";
+
+export default function ProjectsPage() {
+  return (
+    <div>
+      <Banner
+        title="For Nonprofits"
+        text="As a partner, Blueprint will work with you to develop techonology that turns your vision into a reality,
+         completely free of charge!"
+      />
+      <Stack style={{padding: "48px 50px"}} gap="lg">
+        <Container className="top-section"
+        style={{display: "flex",
+                flexDirection: "column",
+                alignItems: "center",
+                justifyContent: "center",
+                color: "#e7f3ff",
+                textAlign: "center"}}>
+          <Title
+          style={{color: "#0078e8",
+                  fontFamily: "Poppins",
+                  fontWeight: 600,
+                  fontSize: "48px",
+                  zIndex: 1,
+                  padding: "10% 0 10%",
+                  lineHeight: "normal"}}>
+            Decision Criteria
+          </Title>
+        </Container>
+        <SimpleGrid
+          cols={2}
+          verticalSpacing={{ base: "md", sm: "sm" }
+          }
+        >
+          <div>
+            <Image
+              h="120%"
+              w="auto"
+              style={{position: "relative",
+                          alignItems: "start",
+                          padding: "6% 10% 20% 20%"}}
+              src="/about_page/placeholder.png" />
+          </div>
+          <div>
+            <Text
+              style={{color: "#0078e8",
+                      fontFamily: "Poppins, sans-serif",
+                      fontSize: 28,
+                      textAlign: "left",
+                      maxWidth: "85%",
+                      paddingTop: "calc(5vw)"}}
+              >
+              Although we want to help as many organizations as
+              possible, we can only work with a few nonprofits at
+              a time because of the amount of resources each
+              partnership requires. As such, we use the following
+              criteria to find projects with the best fit.
+            </Text>
+          </div>
+        </SimpleGrid>
+
+        <SimpleGrid
```

Comment:
These guys get a little squished and overflow when the viewport gets small

Reply:
We can vertically align the items when the viewports get to mobile. We can look at it together